docroot/themes/humsci/humsci_basic/src/scss/tools/_functions.general.scss

```diff
@@ -6,3 +6,29 @@
   $size-in-rems: $size-in-pixels / $hb-root-font-size;
   @return #{$size-in-rems}rem;
 }
+
+// Returns the proper spacing width based on the theme
+@function hb-spacing-width($size: 'default') {
+  $is-valid-theme: false;
+
+  @each $theme in $hb-global-theme-list {
+    @if ($theme == $hb-current-theme) {
+      $is-valid-theme: true;
+    }
+  }
+
+  @if ($is-valid-theme) {
+    $theme-spacing-list: map-get($hb-spacing-list, $hb-current-theme);
+    $result: map-get($theme-spacing-list, $size);
+
+    @if ($result) {
+      // Return spacing value if it exists
+      @return $result;
```

Comment:
hmm there is one instance where the pixel value is needed for grid calculations: on line 31 here: docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.layout.scss

Reply:
Hmm, I would hate for there to be complexity everywhere because of a single exception. Could we have an optional flag like we do in `_mixins.color-pairings.scss` that has a default value and then we can up date whenever we need a px instead of a rem?

```scss
@function hb-spacing-width($size: 'default', $convert-to-rem: true) {
  
  @if ($convert-to-rem) {
    // use hb-calculate-rems and then return rem value
  } @else {
    // return pixel value
  }
}
```

Reply:
that definitely works! thanks Kasey